packages/grid_client/src/modules/stellar.ts

```diff
@@ -276,8 +277,8 @@ class Stellar implements blockchainInterface {
       console.log("Success! View the transaction at: ", transactionResult._links.transaction.href);
       return transactionResult._links.transaction.href;
     } catch (e) {
-      console.log("An error has occured:", e);
-      throw Error(e);
+      console.log("An error has occurred:", e);
```

Comment:
Shouldn't this console log be removed since there's an error being thrown for the same error